packages/terra-popup/src/PopupContent.scss

```diff
@@ -20,20 +22,26 @@
     .close {
       background: var(--terra-popup-content-close-background, 0 0);
       border: var(--terra-popup-content-close-border, 0);
-      color: var(--terra-popup-content-close-color, #111);
       cursor: pointer;
-      font-size: var(--terra-popup-content-close-font-size, 1.45rem);
-      font-weight: var(--terra-popup-content-close-font-weight, bold);
-      line-height: var(--terra-popup-content-close-line-height, 0);
       margin-left: var(--terra-popup-content-close-margin-left, 0.3571rem);
       margin-right: var(--terra-popup-content-close-margin-right, 0);
       padding: var(--terra-popup-content-close-padding, 4px);
+    }
 
-      &:hover {
-        color: var(--terra-popup-content-close-hover-color, #999);
+    .close-icon {
+      background: var(--terra-popup-content-close-icon, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48'><path fill='rgba(17, 17, 17, 1)' d='M28.2 24L42.9 9.1 40.8 7l-1.7-1.6-.4-.5L24 19.7 9.4 4.9 7.2 7 5.6 8.6l-.5.5L19.8 24 5.1 38.9 7.2 41l1.7 1.6.5.5L24 28.3l14.7 14.8.4-.5 1.7-1.6 2.1-2.1L28.2 24z'/></svg>"));
+      display: block;
+      height: 1.25em;
+      width: 1.25em;
+    }
+
+    &:hover {
+      .close-icon {
+        background: var(--terra-popup-content-close-icon, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48'><path fill='rgba(153, 153, 153, 1)' d='M28.2 24L42.9 9.1 40.8 7l-1.7-1.6-.4-.5L24 19.7 9.4 4.9 7.2 7 5.6 8.6l-.5.5L19.8 24 5.1 38.9 7.2 41l1.7 1.6.5.5L24 28.3l14.7 14.8.4-.5 1.7-1.6 2.1-2.1L28.2 24z'/></svg>"));
```

Comment:
close-icon and hover should be switched

````
.close-icon {
  &:hover {
     /* Styles*/
  }
}
````

Otherwise, it will create [dir=ltr]:hover .close-icon, which is what we don't want.

Reply:
There's even another `.close-icon` selector above you can include it on.

Reply:
[Updated](https://github.com/cerner/terra-core/pull/1214/commits/4bbc615c9fe3168235891d7d4ee5cb5c6c181a14).